### PR TITLE
Add more info about how to get started

### PR DESCRIPTION
**Pull request by StratusFearMe21**
The documentation on how to actually *use* this is very sparse, so I added some. Hopefully this helps.

@InvoxiPlayGames (mentioning in case you use the GitHub app, so you can get notified of this PR)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,17 @@ Currently, this server only supports hosting 1 room at a time.
...
+   * If you are running this on a seperate server (not localhost), you should use a domain name and Let's Encrypt to access the server.
+3. If you are running this on a seperate server, edit the top of `johnbox.js` to change accessibleHost to the host you made Let's Encrypt certificates for.
+   * Also, if your certificates are not named localhost.crt and localhost.key, change those also in `johnbox.js`
+5. `node johnbox.js` (or `sudo node johnbox.js` on Linux) to start the server.
+6. Redirect the game to connect to your server. `jbg.config.jet` in each minigame folder has a `serverUrl` parameter.
+   * If your server is running on your local system, put `localhost` in this parameter, if not, put the public URL of the server
+8. On the machine that is running the server, you can go to https://jackbox.tv?server=localhost to play! (Make sure you also have a Browser extension installed that fixes CORS. Just look up CORS on your browser's web store).
```
Comment (InvoxiPlayGames) [inline README.md:58]: This should be fixed in the server software itself, we shouldn't recommend people heavily reduce their browser's security.

File: README.md [hunk 2]
```diff
@@ -42,11 +42,17 @@ Currently, this server only supports hosting 1 room at a time.
 
 ## Usage
 
+*Note*: You must be running the Windows version of the game to use this server. If you are on Linux, use Proton.
```
Comment (InvoxiPlayGames) [inline README.md:45]: Do you know why this is?

File: README.md [hunk 3]
```diff
@@ -42,11 +42,17 @@ Currently, this server only supports hosting 1 room at a time.
...
 1. `npm install ws` to install the WebSockets NodeJS module.
-2. Generate a TLS certificate for the web server to use.
-3. Edit the top of `johnbox.js` to change accessibleHost to a host accessible by all players (e.g. public IP)
-    * This host must have a valid identity in the TLS certificate generated above
-4. `node johnbox` to start the server.
-5. Redirect the game to connect to your server. `jbg.config.jet` in each minigame folder has a `serverUrl` parameter.
\ No newline at end of file
+2. Generate a TLS certificate for the web server to use
+   * You can use [this tutorial](https://gist.github.com/cecilemuller/9492b848eb8fe46d462abeb26656c4f8) to generate these certificates (make sure to use the *Chrome, IE11 & Edge* method when trusting the new certificates, or your OS's specific way).
+   * If you are running this on a seperate server (not localhost), you should use a domain name and Let's Encrypt to access the server.
```
Comment (InvoxiPlayGames) [inline README.md:52]: I don't particularly like the idea of recommending people setting up domain names for this, especially not without a reverse proxy e.g. Nginx.

**Review by InvoxiPlayGames — COMMENTED** [review]
Looks decent, although I have a few concerns with recommendations given here, especially for the experimental nature of the project.

File: README.md (hunk 2 above)
Comment (StratusFearMe21) [reply README.md:45]: Nope

File: README.md (hunk 1 above)
Comment (StratusFearMe21) [reply README.md:58]: I don't see how you could do that, but then again I don't exactly know how CORS works

File: README.md (hunk 3 above)
Comment (StratusFearMe21) [reply README.md:52]: My thinking was that it's the user's choice whether they want to host the server locally or on another server. And if they are hosting on another server, a domain name and a Let's Encrypt certificate is definitely the best way

Comment (InvoxiPlayGames) [reply README.md:52]: It's not.

**Pull request closed without merging**